onnx/common/version.h

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) ONNX Project Contributors.
+// Licensed under the MIT license.
+
+#pragma once
+
+namespace ONNX_NAMESPACE {
+
+// Represents the most recent release version. Updated with every release.
+constexpr const char* LAST_RELEASE_VERSION = "1.7.0";
```

Comment:
Can we read it from: https://github.com/onnx/onnx/blob/master/VERSION_NUMBER?

Reply:
We can't read that file at run-time, right? The one thing we could do is to auto-generate one file from the other. But that seems non-trivial and not worth it (at this point).

Reply:
(Summarizing an offline discussion.) Can we make auto-generation of these files a separate task and do it in a separate PR? It will be better to decide on a solution for all the version-numbers in multiple files, instead of just for this one file.